packages/@okta/vuepress-site/docs/reference/api/domains/index.md

```diff
@@ -0,0 +1,671 @@
+| `domain`                  | String                                                         | Domain name                                                                               | FALSE    |
+| `validationStatus`        | String                                                         | Status of the domain. Values can be `NOT_STARTED`, `IN_PROGRESS`, `VERIFIED`, `COMPLETED` | FALSE    |
+| `dnsRecords`              | [DNSRecords](#dnsrecords-object)                               | TXT and CNAME records to be registered for the domain                                     | FALSE    |
+| `publicCertificate`       | [CertificateMetadata](#certificatemetadata-object)             | Certificate metadata for the domain.                                                               | TRUE     |
+| `_links`                  | [Links](#links-object)                                         | Link relations for this object                                                            | TRUE     |
+
+#### DomainResponse example
+
+```json
+{
+    "id": "OcDz6iRyjkaCTXkdo0g3",
+    "domain": "login.example.com",
+    "validationStatus": "COMPLETED",
+    "dnsRecords": [
+        {
+            "fqdn": "_oktaverification.login.example.com",
+            "values": [
+                "79496f234c814638b1cc44f51a782781"
+            ],
+            "recordType": "TXT"
+        },
+        {
+            "fqdn": "login.example.com",
+            "values": [
+                "https://${yourOktaDomain}.customdomains.okta1.com"
+            ],
+            "recordType": "CNAME"
+        }
+    ],
+    "publicCertificate": {
+        "subject": "CN=login.example.com",
+        "fingerprint": "73:68:82:7B:83:2E:48:29:A5:5E:E8:40:41:80:B3:AA:03:C4:42:43:05:73:45:BC:AA:47:00:23:A3:70:E5:C4",
+        "expiration": "2021-05-11T05:13:05.000Z"
+    },
+    "_links": {
+        "certificate": {
+            "href": "https://${yourOktaDomain}/api/v1/domains/OcDz6iRyjkaCTXkdo0g3/certificate",
+            "hints": {
+                "allow": [
+                    "PUT"
+                ]
+            }
+        },
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/domains/OcDz6iRyjkaCTXkdo0g3",
+            "hints": {
+                "allow": [
+                    "DELETE"
+                ]
+            }
+        },
+        "verify": {
+            "href": "https://${yourOktaDomain}/api/v1/domains/OcDz6iRyjkaCTXkdo0g3/verify",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+### DomainListResponse object
+
+#### DomainListResponse properties
+
+The DomainListResponse object defines several properties:
+
+
+| Property                  | Type                                                           | Description                                                                               | Nullable |
```

Comment:
Remove the Nullable column and start the Description with (Optional) when the property is nullable.
```suggestion
| Property                  | Type                                                           | Description                                                               |
```